tests/driver-no-auto-register.rs

```diff
@@ -1,6 +1,12 @@
-mod utils;
-
 use gdal::{Dataset, DriverManager};
+use std::path::{Path, PathBuf};
+
+/// Returns the fully qualified path to `filename` in `${CARGO_MANIFEST_DIR}/fixtures`.
+fn fixture(filename: &str) -> PathBuf {
+    Path::new(env!("CARGO_MANIFEST_DIR"))
+        .join("fixtures")
+        .join(filename)
+}
```

Comment:
Been considering a non-published `gdal-testkit` module so that we can have shared testing support for unit and integration tests...